src/fixtures/base_fixtures.ts

```diff
@@ -0,0 +1,60 @@
+import { test as base} from '@playwright/test';
+import BasePage from '../../pages/BasePage';
+import BookStorePage from '../../pages/BookStorePage';
+import ButtonsPage from '../../pages/ButtonsPage';
+import ElementsPage from '../../pages/ElementsPage';
+import LoginPage from '../../pages/LoginPage';
+import RegistrationPage from '../../pages/RegistrationPage';
+
+type MyFixture = {
```

Comment:
in ts, when you create types you have to add the letter T before 
```suggestion
type TMyFixture = {
```

same with interfaces, I, ICreds.